x-pack/plugins/saved_objects_tagging/public/management/components/header.tsx

```diff
@@ -24,49 +16,38 @@ interface HeaderProps {
 
 export const Header: FC<HeaderProps> = ({ canCreate, onCreate }) => {
   return (
-    <>
-      <EuiFlexGroup justifyContent="spaceBetween" alignItems="baseline">
-        <EuiFlexItem grow={false}>
-          <EuiTitle>
-            <h1>
-              <FormattedMessage
-                id="xpack.savedObjectsTagging.management.header.title"
-                defaultMessage="Tags"
-              />
-            </h1>
-          </EuiTitle>
-        </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          {canCreate && (
-            <EuiButton
-              key="createTag"
-              iconType="tag"
-              color="primary"
-              fill
-              data-test-subj="createTagButton"
-              onClick={onCreate}
-              isDisabled={false}
-            >
-              <FormattedMessage
-                id="xpack.savedObjectsTagging.management.actions.createTag"
-                defaultMessage="Create tag"
-              />
-            </EuiButton>
-          )}
-        </EuiFlexItem>
-      </EuiFlexGroup>
-      <EuiSpacer size="m" />
-      <EuiText size="s">
-        <p>
-          <EuiTextColor color="subdued">
+    <EuiPageHeader
+      pageTitle={
+        <FormattedMessage
+          id="xpack.savedObjectsTagging.management.header.title"
+          defaultMessage="Tags"
+        />
+      }
+      bottomBorder
+      description={
+        <FormattedMessage
+          id="xpack.savedObjectsTagging.management.header.description"
+          defaultMessage="Use tags to categorize and easily find your objects."
+        />
+      }
+      rightSideItems={[
+        canCreate && (
+          <EuiButton
+            key="createTag"
+            iconType="tag"
+            color="primary"
+            fill
+            data-test-subj="createTagButton"
+            onClick={onCreate}
+            isDisabled={false}
```

Comment:
is this `isDisabled={false}` needed? `false` is the default

Reply:
I'd rather not change the implementation here. Even if `isDisabled` is `false` by default, there's no harm in specifying it for clarity.